recipes/milksnake/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "milksnake" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: dfcd43b78bcf93897a75eea1dadf71c848319f19451cff4f3f3a628a5abe1688
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - cffi
+  run:
+    - cffi
+    - python
+
+test:
+  imports:
+    - milksnake
+    - milksnake.ffi
+
+about:
+  home: http://github.com/simplejson/simplejson
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: 'Milksnake is an extension for setuptools that allows you to distribute dynamic linked
+            libraries in Python wheels in the most portable way imaginable.'
+  doc_url: https://github.com/getsentry/milksnake/blob/master/README.md
+  dev_url: https://github.com/getsentry/milksnake
+
+extra:
+  recipe-maintainers:
+    - pacman82
+    - xochy
```

Comment:
Ups, you mistyped my author name ;)

Reply:
🤦‍♂ , but I see you fixed it already in the conda-forge repository.